### ISSUE TITLE
mask rcnn on unseen dataset

### ISSUE THREAD

**Issue by ssetty** [matterport/Mask_RCNN#837]
Execution of detection or prediction on unseen dataset yields no bounding mask. However if I take sample from train/validation set for some images bounds are detected. What could be reason - is it overfitting or lack of training samples or anchors needs to be adjusted?

Thanks
